src/H5VL_log_file.cpp

```diff
@@ -173,6 +173,10 @@ void *H5VL_log_file_create (
         H5VL_LOGI_PROFILING_TIMER_STOP (fp, TIMER_H5VLFILE_CREATE);
         H5VL_LOGI_PROFILING_TIMER_STOP (fp, TIMER_H5VL_LOG_FILE_CREATE_FILE);
         H5VL_LOGI_PROFILING_TIMER_STOP (fp, TIMER_H5VL_LOG_FILE_CREATE);
+        if (under_vol_info == NULL) {
```

Comment:
It might be less fragile just to start by setting `uvlid` to `H5I_INVALID_HID` and then checking if it's still that rather than assuming `under_vol_info == NULL` means `uvlid` was set.

Reply:
Updated to use the `info` state